src/schemas/config.schema.json

```diff
@@ -151,7 +151,11 @@
         },
         "geneFilter": {
           "type": "string",
-           "description": "The location in the AnnData store of a filter for the genes if using a subset of the data, like 'vae.highly_variable.' if the matrix comes from 'obsm/hvg_subset.'"
+           "description": "The location in the AnnData store of a filter for the genes if using a subset of the data, like 'var.highly_variable.' if the matrix comes from 'obsm/hvg_subset.'"
+        },
+        "heatmapFilter": {
```

Comment:
What's the difference between `geneFilter` and `heatmapFilter`? Is `geneFilter` supposed to tell us, out of the full expression matrix `X`, which genes are included as the columns of the potentially smaller matrix specified in `matrix`? And then `heatmapFilter` tells us, out of those genes in `matrix`, which should be loaded+rendered for the heatmap initially?